packages/app/src/systems/Transaction/components/TxContent/TxContent.tsx

```diff
@@ -156,7 +158,10 @@ function TxContentInfo({
         isLoading={isLoading}
       />
       {isLoading && !showDetails && <TxFee.Loader />}
-      {showDetails && !fees && <TxFee fee={tx?.fee} />}
+      {showDetails && !fees && (
+        // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+        <TxFee fee={tx?.fee} tipInUsd={(tx as any)?.feeInUsd} />
```

Comment:
instead of passing this "inUsd" all the time, can we just have a function that will convert it everywhere?

like 
```
convertToUsd(amount, rate, decimals)
```